lib/view_models/user_list/post_hos_list_view_model.dart

```diff
@@ -13,6 +13,7 @@ class PostHosViewModel {
     if (postHosList != null) {
       postHosList.forEach((mapData) {
         userList.add(PostHosData(map: mapData));
+        userList.sort((a, b) => b.status.compareTo(a.status));
```

Comment:
use the sort function in its ownclass might be better มั้ง idk

Reply:
resolved https://github.com/LaCare-KMUTT/AbdoCare-Web/pull/75/commits/dedc0ff4112437eb8b7b263eed137cce8bc71b1e